examples/experimental/gentoo.yaml

```diff
@@ -0,0 +1,8 @@
+images:
+- location: "https://distfiles.gentoo.org/experimental/amd64/openstack/gentoo-openstack-amd64-default-latest.qcow2"
+  arch: "x86_64"
+
+# The built-in containerd installer does not support Gentoo currently.
+containerd:
+  system: false
+  user: false
```

Comment:
Can you add some comment about sshfs?

Reply:
In "theory" it should work, but the system upgrade process itself failed...

But it is not included in the default image, so will just add empty mounts and a comment.

Haven't tried 9p, either

Possible that it just works

Reply:
The default kernel works fine with 9p, it has support builtin (as a module)